battery-calcuator/index.html

```diff
@@ -0,0 +1,508 @@
+                        <span class="bc-unit"></span>
+                    </div>
+                </div>
+                <div class="bc-output-row">
+                    <div class="bc-column bc-col-label">Receive/idle duration</div>
+                    <div class="bc-column bc-col-value">
+                        <span class="bc-value" v-html="updatedOutputText(resultValues.rxduration)"></span>
+                        <span class="bc-unit"></span>
+                    </div>
+                </div>
+                <div class="bc-output-row">
+                    <div class="bc-column bc-col-label">Sleep duration</div>
+                    <div class="bc-column bc-col-value">
+                        <span class="bc-value" v-html="updatedOutputText(resultValues.sleepduration)"></span>
+                        <span class="bc-unit"></span>
+                    </div>
+                </div>
+            </div>
+
+            <div class="bc-output-group">
+                <div class="bc-output-row">
+                    <div class="bc-column bc-col-label">Transmit battery capacity requirement</div>
+                    <div class="bc-column bc-col-value">
+                        <span class="bc-value">[[ resultValues.txcapacity ]]</span>
+                        <span class="bc-unit">Wh</span>
+                    </div>
+                </div>
+                <div class="bc-output-row">
+                    <div class="bc-column bc-col-label">Receive battery capacity requirement</div>
+                    <div class="bc-column bc-col-value">
+                        <span class="bc-value">[[ resultValues.rxcapacity ]]</span>
+                        <span class="bc-unit">Wh</span>
+                    </div>
+                </div>
+                <div class="bc-output-row">
+                    <div class="bc-column bc-col-label">Sleep battery capacity requirement</div>
+                    <div class="bc-column bc-col-value">
+                        <span class="bc-value">[[ resultValues.sleepcapacity ]]</span>
+                        <span class="bc-unit">wh</span>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+
+        <div class="disclaimer">
+            <p><b>Disclaimer:</b> The battery capacity calculation tool for SWIS - ADCP edition offers a general estimation based on assumptions regarding link speed and power consumption for subsea deployments. However, if the underwater channel conditions differ significantly from typical channels used for these calculations, the assumptions may not be accurate, leading to incorrect calculations. The tool should be used as guidance rather than definitive values, and users are advised to exercise their own judgment, conduct further analysis, and consult experts. The developers and providers of the tool are not liable for any damages or losses resulting from its use or reliance on its calculations.</p>
```

Comment:
```suggestion
            <p><b>Disclaimer:</b> The battery capacity calculation tool for SWIS - ADCP edition offers an estimate based on assumptions regarding link speed and power consumption for subsea deployments. However, if the underwater channel conditions differ significantly from typical channels used for these calculations, the assumptions may not be accurate and thus leading to incorrect calculations. The tool should be used as guide rather than to obtain definitive values. Users are advised to exercise their own judgment, conduct further analysis, and consult experts. The developers and providers of the tool are not liable for any damages or losses resulting from its use or reliance on its calculations.</p>
```